x-pack/plugin/otel-data/src/main/resources/index-templates/metrics-otel@template.yaml

```diff
@@ -0,0 +1,25 @@
+---
+version: ${xpack.oteldata.template.version}
+index_patterns: ["metrics-*.otel-*"]
+priority: 120
+data_stream: {}
+allow_auto_create: true
+_meta:
+  description: default OpenTelemetry metrics template installed by x-pack
+  managed: true
+composed_of:
+  - metrics@tsdb-settings
+  - otel@mappings
```

Comment:
Let's be consistent with the ordering of whether settings or mappings are listed first. Also have a look at the index templates in core to ensure consistency with them.

Reply:
IIRC, in the apm-data plugin, there are unit tests that verify the consistency of the different index templates. It probably makes sense to adapt them for this plugin, too. Feel free to do this as a follow-up.

Reply:
Seems it's always first mappings and then settings - so I aligned it. I'll defer the unit tests to a follow up PR - or if we need to wait longer for a review here I may push it here - it's on my todo list.